aspects/request_timeout_handler.go

```diff
@@ -0,0 +1,41 @@
+package aspects
```

Comment:
Please consider moving to the `router` package to co-locate with router setup or to `endpoints` package to co-locate with the handlers.. or perhaps maybe `endpoints/aspects`?

Reply:
Moved to router package